src/event.rs

```diff
@@ -74,9 +74,93 @@ pub enum Event<'a, T: 'static> {
     UserEvent(T),
 
     /// Emitted when the application has been suspended.
+    ///
+    /// # Portability
+    ///
+    /// Not all platforms support the notion of suspending applications, and there may be no
+    /// technical way to guarantee being able to emit a `Suspended` event if the OS has
+    /// no formal application lifecycle (currently only Android and iOS do). For this reason
+    /// Winit does not currently try to emit pseudo `Suspended` events before the application
+    /// quits on platforms without an application lifecycle.
+    ///
+    /// In case of corner cases with how platform-specific events are mapped to Winit `Suspended`
+    /// and `Resumed` events it's recommended that applications be able to gracefully handle redundant
+    /// (i.e. back to back) `Suspended` or `Resumed` events.
+    ///
+    /// Also see [`Resumed`][Self::Resumed] notes.
+    ///
+    /// # Android
```

Comment:
Since this fits under the `Portability` umbrella, should we make this h2 with two `##`?